stablehlo/dialect/StablehloOps.td

```diff
@@ -2140,11 +2140,8 @@ def StableHLO_CustomCallOp: StableHLO_Op<"custom_call",
 
     Example:
     ```mlir
-    %results = "stablehlo.custom_call"(%input0) {
-      call_target_name = "foo",
-      has_side_effect = false,
+    %results = stablehlo.custom_call @foo(%input0) {
```

Comment:
Can you check if the pretty printed version is round trippable to preserve the attributes like `api_version` and `has_side_effects`?

Reply:
Yes, it will keep all of the relevant info and checked with this command:
```
stablehlo-opt file.mlir | stablehlo-opt
```
Though it will drop values that are already default.